Userland/Libraries/LibTest/TestCase.h

```diff
@@ -8,16 +8,36 @@
 #pragma once
 
 #include <LibTest/Macros.h> // intentionally first -- we redefine VERIFY and friends in here
+#include <LibTest/Randomized/RandomnessSource.h>
+#include <LibTest/Randomized/Shrink.h>
 
 #include <AK/DeprecatedString.h>
 #include <AK/Function.h>
 #include <AK/NonnullRefPtr.h>
 #include <AK/RefCounted.h>
 
+#ifndef MAX_GENERATED_VALUES_PER_TEST
+#    define MAX_GENERATED_VALUES_PER_TEST 100
+#endif
+
+#ifndef MAX_GEN_ATTEMPTS_PER_VALUE
+#    define MAX_GEN_ATTEMPTS_PER_VALUE 15
+#endif
```

Comment:
These look like they simply might want to be local `static constexpr size_t` or something like that.

Reply:
How do we override them later though? Do we want to make the test runner accept a cmdline flag?

Reply:
If they are supposed to be changeable, then `constexpr` really isn't the right choice either.

But in any case, needing to change these at build time by manually editing build flags to pass definitions is weird, if they are meant to be changed often. If they aren't meant to be changed often, then having them as a `constexpr` value that has to be edited in the source code is probably fine, certainly for a first pass.

Reply:
It's useful in practice to have easy access to changing `MAX_GENERATED_VALUES_PER_TEST`. Sometimes you want to run eg. 10k tries instead of 100, just to get extra confidence.

In comparison, I've never needed to change `MAX_GEN_ATTEMPTS_PER_VALUE` and that one perhaps could be a constexpr.

I'll make the first one a runtime flag (similarly to how `--benchmark_repetitions N` is done) and the second one a constexpr.